memcached-operator/README.md

```diff
@@ -53,50 +43,30 @@ $ operator-sdk build $IMAGE
 $ docker push $IMAGE
 ```
 
-### Run the tests
-
-To run the tests in `test/e2e`, first make sure to update the image name from `REPLACE_IMAGE`
-to your desired image name in `deploy/operator.yaml`:
+### Using the image
 
 ```
-$ sed "s@REPLACE_IMAGE@$IMAGE@g" -i deploy/operator.yaml
+# Update the operator manifest to use the built image name (if you are performing these steps on OSX, see note below)
+$ sed -i 's|REPLACE_IMAGE|quay.io/example-inc/memcached-operator|g' deploy/operator.yaml
+# On OSX use:
+$ sed -i "" 's|REPLACE_IMAGE|quay.io/example-inc/memcached-operator|g' deploy/operator.yaml
 ```
 
-Run the tests that reside in `test/e2e`:
+**NOTE** The `quay.io/example-inc/memcached-operator` is an example. You should build and push the image for your repository. 
 
-```
-$ operator-sdk test local ./test/e2e
-```
+### Installing
 
-To run go-test with verbose and limit to 2 parallel tests:
+Run `make install` and check the operator and the example Memcached deployed and running in the cluster 
 
-```
-$ operator-sdk test local ./test/e2e --go-test-flags "-v -parallel=2"
-```
+### Uninstalling 
 
-To run the tests from an image in the cluster, first build the tests into your image:
+To uninstall all that was performed in the above step run `make uninstall`.
 
-```
-$ operator-sdk build $IMAGE --enable-tests
-$ docker push $IMAGE
-```
-
-Then, configure the testing environment:
-
-```
-$ export NAMESPACE="my-test"
-$ kubectl create -f deploy/crds/cache_v1alpha1_memcached_crd.yaml -n $NAMESPACE
-$ kubectl create -f deploy/service_account.yaml -n $NAMESPACE
-$ kubectl create -f deploy/role.yaml -n $NAMESPACE
-$ kubectl create -f deploy/role_binding.yaml -n $NAMESPACE
-```
-
-Finally, run the tests:
-```
-$ operator-sdk test cluster $IMAGE --namespace $NAMESPACE --service-account memcached-operator
-```
+**NOTE:** To check what is done in the Make commands see the [Makefile](./Makefile)
```

Comment:
I would remove this line entirely. That's the first thing I would do is see what is in the `Makefile` :) Also, I have a way to add `help` to the `Makefile` so when you run `make help` it prints out all the targets and a brief help message with them.

Here's the help from the Ansible Service Broker's Makefile:
```
[jesusr@transam ansible-service-broker{release-1.2}]$ make help
Usage: make <OPTIONS> ... <TARGETS>

Available targets are:

vendor               Install or update project dependencies
broker               Build the broker
build                Build binary from source
generate             regenerate mocks
lint                 Run golint
fmt                  Run go fmt
fmtcheck             Check go formatting
test                 Run unit tests
test-coverage-html   checkout the coverage locally of your tests
ci-test-coverage     CI test coverage, upload to coveralls
vet                  Run go vet
check                Pre-flight checks before creating PR
prep-local           Prepares the local dev environment
build-image          Build a docker image with the broker binary
release              Builds docker container using latest rpm from Copr
clean                Clean up your working environment
really-clean         Really clean up the working environment
deploy               Deploy a built broker docker image to a running cluster
cleanup-ci           Cleanup after ci run
ci                   Run the CI workflow locally
help                 Show this help screen
```

Reply:
It makes sense. Following the help added.  

<img width="904" alt="Screenshot 2019-09-18 at 21 23 30" src="https://user-images.githubusercontent.com/7708031/65183130-b9c33280-da5a-11e9-8c09-a8189a56adc2.png">

Reply:
c/c @theishshah ^